### DISCUSSION TITLE
next12 How to turn it on http2

### DISCUSSION THREAD

**Discussion by lynn1286** [vercel/next.js#44154] [Help]
### Describe the feature you'd like to request

![image](https://user-images.githubusercontent.com/40122311/208448122-ea6a890f-deab-4491-a2d3-b94a51aee982.png)

I have a question, nextjs is not h2 when requested by the server. What's the problem? My website is deployed through docker. nginx has configured h2. All requests from the client are h2, but the server is not h2. I don't know what caused the problem. Can you answer my questions? Thank you very much。

I checked some information and said that next supports h2 out of the box, so what should I do to use this function?

### Describe the solution you'd like

![image](https://user-images.githubusercontent.com/40122311/208449414-2566fb61-cf9b-4aaf-b60f-b130dbd8911d.png)

### Describe alternatives you've considered

no

**Comment by powerfulyang** [comment]
What is your nginx conf file?

**Reply by powerfulyang** [reply]
https://github.com/vercel/next.js/tree/canary/examples/with-http2
Enable nginx http2, use `proxy_pass` directive to point the next.js service.
